build_irods_packages.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+# Compile the GenQuery2 project.
+git clone https://github.com/irods/irods_api_plugin_genquery2
+mkdir /_build_genquery2
+cd /_build_genquery2
+cmake -GNinja -DIRODS_ENABLE_430_COMPATIBILITY=NO /irods_api_plugin_genquery2
+ninja package
+apt install -y ./*.deb
```

Comment:
Avoid use of `apt` in scripts. Instead, use `apt-get` and `dpkg`

Reply:
Done.